src/main/proto/embedding_gateway.proto

```diff
@@ -167,6 +167,7 @@ message GetSupportedProvidersResponse {
       map<string, string> properties = 4;
       optional bool deprecated = 5;
       optional string serviceUrl = 6;
+      string display_name = 7;
```

Comment:
Should this be optional for backwards compatibility?

Reply:
Added display name to all the models in the configuration by copying the model name.